postgres-connector/build.gradle

```diff
@@ -2,16 +2,25 @@
  * SPDX-License-Identifier: Apache-2.0
  * Copyright Contributors to the ODPi Egeria project.
  */
+apply plugin: 'java'
 
+//repositories {
+  //  maven {
+    //    url("https://mvnrepository.com/artifact/org.postgresql/postgresql")
+      //  }
+  //}
+
+ configurations {
+    releaseJars
+}
 
 dependencies {
-    implementation 'org.junit.jupiter:junit-jupiter:5.4.2'
+    compile group: 'org.postgresql', name: 'postgresql', version: '42.1.4'
```

Comment:
Cool. Good way of doing it - something we need to be better at in the base (using packaging BOMs basically)